### PR TITLE
feat: Remove `iui-textarea`

### PR DESCRIPTION
**Pull request by mayank99**
`iui-textarea` only has `resize` rules in addition to `iui-input`, so it doesn't need to be a separate component. I've moved the resize declaration to `iui-input`, it will have no effect when used on a `<input>` element.

### CODE REVIEW

File: packages/itwinui-css/src/input/input.scss [hunk 1]
```diff
@@ -57,6 +57,8 @@
     cursor: not-allowed;
     --_hover-color: var(--iui-color-background-disabled);
   }
+
+  @include textarea;
```
Comment (veekeys) [inline packages/itwinui-css/src/input/input.scss:61]: Can we apply this only for `textarea` elements? Looks invalid, when we have `resize` prop on `input`.
We can use your loved `:where` to keep specificity down.

Comment (veekeys) [reply packages/itwinui-css/src/input/input.scss:61]: Trash in css, which is not needed

Comment (mayank99) [reply packages/itwinui-css/src/input/input.scss:61]: This looks like a lot more "trash" to me


but changed

Comment (veekeys) [reply packages/itwinui-css/src/input/input.scss]: Something got messed up with lock file?

Comment (mayank99) [reply packages/itwinui-css/src/input/input.scss]: i just ran `yarn` when i merged v1 branch here. maybe it was messed up in the v1 branch

Comment (mayank99) [reply packages/itwinui-css/src/input/input.scss]: pushed updated lockfile in v1 branch so it should not show changed here anymore

**Review by veekeys — APPROVED** [review]

**Pull request merged**